src/main/groovy/com/wooga/gradle/io/OutputStreamSpec.groovy

```diff
@@ -20,26 +25,118 @@ trait OutputStreamSpec extends BaseSpec {
         logToStdout
     }
 
+    private final Property<Boolean> logToStderr = objects.property(Boolean)
+
+    /**
+     * @return Whether the process should also log to stderr
+     */
+    @Internal
+    Property<Boolean> getLogToStderr() {
+        logToStderr
+    }
+
     /**
      * @return An output stream that can fork to a given logFile, if it's been set
      */
     OutputStream getOutputStream(File logFile) {
```

Comment:
This and the other getters for stream aren't getters, are factories/creators. As we are inside a trait we have to be extra careful with naming getters, as we can have a property called "outputStream" on the parent classes, or in other traits that can overlap with this. So I believe it would be better to rename it to "createOutputStream", "createStandardOutputStream", etc.

I know this sounds quite picky but I believe it can get us into trouble in the future

Reply:
I understand; these are to configure what the output stream spec does. 
Perhaps in the future we may shift these.